hikari/impl/entity_factory.py

```diff
@@ -1469,8 +1470,9 @@ def _set_invite_attributes(self, payload: data_binding.JSONObject) -> _InviteFie
         else:
             channel_id = snowflakes.Snowflake(payload["channel_id"])
 
-        target_user_type = (
-            invite_models.TargetUserType(payload["target_user_type"]) if "target_user_type" in payload else None
+        target_type = invite_models.TargetType(payload["target_type"]) if "target_type" in payload else None
+        target_application = (
+            self.deserialize_application(payload["target_application"]) if "target_application" in payload else None
```

Comment:
Isn't this a partial application object? Also isn't this feature unreleased rn?

Reply:
It has been documented already: https://github.com/discord/discord-api-docs/commit/1b4e363e324eb1f49a47e32cb0108fbe276c8e0e

And we currently don't have a proper PartialApplication object, so idk what they send and they don't

Reply:
> It has been documented already: discord/discord-api-docs@1b4e363

it's still unreleased though, wouldn't be the first time discord's (let someone) document stuff too early then had to remove it or made breaking changes and we do have a PartialApplication just it only exists for other application objects to subclass off of and isn't actually returned anywhere itself so it wouldn't be appropriate to just slap that here

Reply:
> it's still unreleased though, wouldn't be the first time discord's (let someone) document stuff too early then had to remove it or made breaking changes and we do have a PartialApplication just it only exists for other application objects to subclass off of and isn't actually returned anywhere itself so it probably wouldn't be appropriate to just slap that here

I meant not proper due to it not being exposed in the `__all__` of guilds.py. We can keep this pr on hold until Discord implements it